### ISSUE TITLE
docfx@2.58.8: decompress error

### ISSUE THREAD

**Issue by m4ss1m0g** [ScoopInstaller/Main#2882]
Extracting docfx.zip ... ERROR Exit code was 2!
Failed to extract files from C:\Users\xxxxx\scoop\apps\docfx\2.58.8\docfx.zip.
Log file:
  ~\scoop\apps\docfx\2.58.8\7zip.log

## 7zip.log 

7-Zip 19.00 (x64) : Copyright (c) 1999-2018 Igor Pavlov : 2019-02-21

Scanning the drive for archives:
1 file, 21335883 bytes (21 MiB)

Extracting archive: C:\Users\xxxxx\scoop\apps\docfx\2.58.8\docfx.zip

WARNINGS:
Headers Error

--
Path = C:\Users\xxxx\scoop\apps\docfx\2.58.8\docfx.zip
Type = zip
ERRORS:
Unexpected end of archive
WARNINGS:
Headers Error
Physical Size = 1056981597
Characteristics = Local

Sub items Errors: 1

Archives with Errors: 1

Warnings: 1

Open Errors: 1

Sub items Errors: 1

**Comment by tech189** [comment]
I think this has been fixed with the update to version 2.58.9 in this commit: https://github.com/ScoopInstaller/Main/commit/219e5decfd48e381eb807ede47504cd75bd893c8
```
❯ scoop install docfx
Installing 'docfx' (2.58.9) [64bit]
docfx.zip (21.5 MB) [=========================================================================================] 100%
Checking hash of docfx.zip ... ok.
Extracting docfx.zip ... done.
Linking ~\scoop\apps\docfx\current => ~\scoop\apps\docfx\2.58.9
Creating shim for 'docfx'.
'docfx' (2.58.9) was installed successfully!
```

**Comment by rashil2000** [comment]
Hey @tech189!

Thanks for the heads up.

@m4ss1m0g please try downloading the latest version.
